utils/cliutils/commandsflags.go

```diff
@@ -0,0 +1,1223 @@
+	},
+	"go": {
+		noRegistry, publishDeps, deprecatedUrl, deprecatedUser, deprecatedPassword, deprecatedApikey,
+		deprecatedAccessToken, buildName, buildNumber, module,
+	},
+	"go-recursive-publish": {
+		url, user, password, apikey, accessToken, serverId,
+	},
+	"ping": {
+		url, user, password, apikey, accessToken, sshPassPhrase, sshKeyPath, serverId, clientCertPath,
+		clientCertKeyPath, insecureTls,
+	},
+	"curl": {
+		serverId,
+	},
+	"pip-config": {
+		global, serverIdResolve, repoResolve,
+	},
+	"pip-install": {
+		buildName, buildNumber, module,
+	},
+	"release-bundle-create": {
+		url, distUrl, user, password, apikey, accessToken, sshKeyPath, sshPassPhrase, serverId, spec, specVars,
+		rbDryRun, sign, desc, exclusions, releaseNotesPath, releaseNotesSyntax, rbPassphrase, rbRepo, insecureTls,
+	},
+	"release-bundle-update": {
+		url, distUrl, user, password, apikey, accessToken, sshKeyPath, sshPassPhrase, serverId, spec, specVars,
+		rbDryRun, sign, desc, exclusions, releaseNotesPath, releaseNotesSyntax, rbPassphrase, rbRepo, insecureTls,
+	},
+	"release-bundle-sign": {
+		url, distUrl, user, password, apikey, accessToken, sshKeyPath, sshPassPhrase, serverId, rbPassphrase, rbRepo,
+		insecureTls,
+	},
+	"release-bundle-distribute": {
+		url, distUrl, user, password, apikey, accessToken, sshKeyPath, sshPassPhrase, serverId, rbDryRun, distRules,
+		site, city, countryCodes, sync, maxWaitMinutes, insecureTls,
+	},
+	"release-bundle-delete": {
+		url, distUrl, user, password, apikey, accessToken, sshKeyPath, sshPassPhrase, serverId, rbDryRun, distRules,
+		site, city, countryCodes, sync, maxWaitMinutes, insecureTls, deleteFromDist, deleteQuiet,
+	},
+	"template-consumer": {
+		url, user, password, apikey, accessToken, sshPassPhrase, sshKeyPath, serverId, clientCertPath,
+		clientCertKeyPath, vars,
+	},
+	"repo-delete": {
+		url, user, password, apikey, accessToken, sshPassPhrase, sshKeyPath, serverId, clientCertPath,
+		clientCertKeyPath, deleteQuiet,
+	},
+	"replication-delete": {
+		url, user, password, apikey, accessToken, sshPassPhrase, sshKeyPath, serverId, clientCertPath,
+		clientCertKeyPath, deleteQuiet,
+	},
+	"access-token-create": {
+		url, user, password, apikey, accessToken, sshPassPhrase, sshKeyPath, serverId, clientCertPath,
+		clientCertKeyPath, groups, grantAdmin, expiry, refreshable, audience,
+	},
+	// Xray's commands
+	"offline-update": {
+		licenseId, from, to, version, target,
+	},
+	// Mission Control's commands
+	"mc-config": {
+		mcUrl, mcAccessToken, mcInteractive,
+	},
+	"license-acquire": {
+		mcUrl, mcAccessToken,
+	},
+	"license-deploy": {
+		mcUrl, mcAccessToken, licenseCount,
+	},
+	"license-release": {
+		mcUrl, mcAccessToken,
+	},
+	"jpd-add": {
+		mcUrl, mcAccessToken,
+	},
+	"jpd-delete": {
+		mcUrl, mcAccessToken,
+	},
+}
+
+func GetCommandFlags(cmd string) []cli.Flag {
+	flagList, ok := commandFlags[cmd]
+	if !ok {
+		return nil
```

Comment:
I suggest adding an error log message before returning nil, with the cmd value sent.